app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -161,19 +237,51 @@ class StateFragmentTest {
     launchForExploration(TEST_EXPLORATION_ID_30).use {
       startPlayingExploration()
       onView(withId(R.id.continue_button)).perform(click())
-      onView(withId(R.id.fraction_input_interaction_view)).perform(typeText("1/"))
+      onView(withId(R.id.fraction_input_interaction_view)).perform(
+        typeText("1/"),
+        closeSoftKeyboard()
+      )
+      onView(withId(R.id.state_recycler_view)).perform(scrollToPosition<RecyclerView.ViewHolder>(2))
+      onView(withId(R.id.submit_answer_button)).perform(click())
+
+      // Add another '2' to change the pending input text.
+      onView(withId(R.id.fraction_input_interaction_view)).perform(
+        typeText("2"),
+        closeSoftKeyboard()
+      )
+
+      // The submit button should be re-enabled since the text view changed.
+      onView(withId(R.id.submit_answer_button)).check(matches(isClickable()));
```

Comment:
Remove semicolon from this line.

Reply:
Removed here and everywhere else in this file.